src/components/search/styles.css

```diff
@@ -24,12 +24,24 @@
 
 .search-root .hit {
   @apply flex items-center;
+  /* Avoid having search results displayed on multiple lines */
+  height: 2em;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  width: 100%;
 }
 
 .search-root .hit a {
   @apply flex-grow;
 }
 
+@media (max-width: 768px) {
+  .no-results {
+    text-align: center;
+  }
+}
+
```

Comment:
I think this can be done in tailwinds 

https://tailwindcss.com/docs/text-align/#app

`sm:text-center`

I assume most properties have an `sm:` prefix

Reply:
using sm would not work for xs, that is why I used media queries. 

The better solution with tailwind is to use a global `text-center` and specify for bigger devices with `md:text-left` :)

Reply:
good call!

Reply:
My mistake was to not think "mobile-first". So I made a specific code for mobile while we should do a specific code for desktop. Tailwind makes us enforce this philosophy.
I ended up using media query because I didn't understand that at the time and couldn't figure why xs:* didn't work. 
Tailwind simply does not have a xs* because we should use the default code for mobile and then use md* for desktop

Reply:
Good point. I was not thinking that way either.